content_scripts/mode_find.coffee

```diff
@@ -178,7 +178,10 @@ class FindMode extends Mode
       # ignore the selectionchange event generated by find()
       document.removeEventListener("selectionchange", @restoreDefaultSelectionHighlight, true)
 
-    result = window.find(query, options.caseSensitive, options.backwards, true, false, true, false)
+    try
+      result = window.find(query, options.caseSensitive, options.backwards, true, false, true, false)
+    catch # Failed searches throw on Firefox.
+    options.postFindFocus?.focus()
```

Comment:
Is the indentation correct here?  Looks like `XXX.focus()` should happen only if the exception is raised.